network/nxos/nxos_vxlan_vtep_vni.py

```diff
@@ -78,6 +78,26 @@
         required: false
         default: present
         choices: ['present','absent']
+    include_defaults:
```

Comment:
@privateip Are we standardising on `defaults:` or `include_defaults:` (Assuming this is the same thing)